File: app/src/main/java/jp/java_conf/ikastyle/Entity/ClothingGear.java
package jp.java_conf.ikastyle.Entity;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;

@Entity(tableName = "MAST_CLOTHING_GEAR", primaryKeys = {"id", "language_code"})
public class ClothingGear {
    @ColumnInfo(name = "id")
    public int id;

    @ColumnInfo(name = "language_code")
    public int languageCode;

    @NonNull
    @ColumnInfo(name = "name")
    public String name;
}

File: app/src/main/java/com/splatool/ikastyle/entity/ClothingGear.kt
package com.splatool.ikastyle.entity

import androidx.annotation.NonNull
import androidx.room.*

@Entity(tableName = "MAST_CLOTHING_GEAR", primaryKeys = ["id", "language_code"])
data class ClothingGear (
    @ColumnInfo(name = "id") val id: Int,
    @ColumnInfo(name = "language_code") val languageCode: Int,
    @ColumnInfo(name = "name") @NonNull val name: String
)